packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -97,6 +97,21 @@ export class Loan {
   async setCollateralToken (collateralToken: SetCollateralToken, utxos: UTXO[] = []): Promise<string> {
     return await this.client.call('setcollateraltoken', [collateralToken, utxos], 'number')
   }
+
+  /**
+   * Creates a vault transaction.
+   *
+   * @param {CreateVault} vault
+   * @param {string} vault.ownerAddress Any valid address or "" to generate a new address
```

Comment:
not sure why this is the way it is. instead of passing "", it could be made optional.

Reply:
@jingyi2811 please reference this as well when you list inconsistencies